### PR TITLE
Move hints and attributes into the right folder

### PR DESCRIPTION
**Pull request by martijn00**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Cleanup

### :arrow_heading_down: What is the current behavior?

Not the same folder used as the other platforms

### :boom: Does this PR introduce a breaking change?

Yes a bit.

### :thinking: Checklist before submitting

- [ ] All projects build
- [ ] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [ ] Rebased onto current develop

### CODE REVIEW

File: MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs [hunk 1]
```diff
@@ -202,6 +203,15 @@ public override void RegisterAttributeTypes()
                 }
                 return;
             }
+            if (hint is MvxPopRecursivePresentationHint popRecursiveHint) {
+                var levels = popRecursiveHint.LevelsDeep;
+                if (levels > navigation.NavigationStack.Count())
+                    levels = navigation.NavigationStack.Count();
```
Comment (nickrandolph) [inline MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs]: @martijn00 shouldn't this be count-1. If levels==Count, won't it attempt to pop off all pages?

Comment (martijn00) [reply MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs]: Yes it will pop it all off. It's up to the user if they want that, and to control it.

Comment (nickrandolph) [reply MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs]: In that case should we be doing the check for levels > count at all? I think popping the last page fails doesn't it?

Comment (martijn00) [reply MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs]: I'm not sure if popping the last page fails. Popping more than that should not work in any case i guess.

**Pull request merged**